pyronearEngine/inference/__init__.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) Pyronear contributors.
+# This file is dual licensed under the terms of the CeCILL-2.1 and AGPLv3 licenses.
+# See the LICENSE file in the root of this repository for complete details.
+
+from .pyronearPredict import *
```

Comment:
if you do `import *` , then please add `__all__ = ['PyronearPredictor']` after the imports of pyronearEngine/inference/pyronearPredict.py (it will limit the imported objects)

Reply:
ok I made a classic import it is simpler considering there is only one import here